test/comparators_test.go

```diff
@@ -0,0 +1,58 @@
+package test_test
+
+import (
+	"fmt"
+	"errors"
+	"testing"
+
+	"github.com/LUSHDigital/core/test"
+	"github.com/google/go-cmp/cmp"
+)
+
+type TestError struct{ error }
+
+func TestErrorTypeComparator(t *testing.T) {
+	testErr := TestError{errors.New("oops")}
+	someErr := errors.New("ouch")
+	opts := cmp.Options{
+		test.ErrorTypeComparator,
+	}
+	if cmp.Equal(testErr, someErr, opts) {
+		t.Fatalf("exected unequality between %T and %T\n", someErr, testErr)
```

Comment:
This seems like some really obtuse language. `exected` is a strange choice of word. Did you mean expected? If so, I would suggest saying `expected %T and %T to be of the same type`.

Reply:
I did mean `expected`. I've corrected this.
The point here though is to ensure both types are **not** the same.
I've also corrected that wording.